### PR TITLE
[1LP][RFR] Removed older option and simplified codebase

### PR DESCRIPTION
**Pull request by psav**
Removing an older option that is no longer present in 5.6+
Removed get_yaml_file() as it is no longer used

### CODE REVIEW

File: utils/appliance/__init__.py [hunk 1]
```diff
@@ -1931,64 +1927,48 @@ def host_id(self, hostname):
...
+        temp_ruby = load_data_file(ruby_template.strpath, ruby_replacements)
+        self.ssh_client.put_file(temp_ruby.name, dest_ruby)
+
+        # Run it
+        if self.ssh_client.run_rails_command(dest_ruby):
+            self.server_details_changed()
         else:
-            # 2 = errno.ENOENT
-            raise IOError(2, 'Remote yaml not found or permission denied')
+            raise Exception('Unable to set config')
```
Comment (izapolsk) [inline utils/appliance/__init__.py]: OPTIONAL: "ApplianceException" ?

File: utils/appliance/__init__.py [hunk 2]
```diff
@@ -1931,64 +1927,48 @@ def host_id(self, hostname):
...
+    def get_yaml_config(self):
+        writeout = self.ssh_client.run_rails_command(
+            '"File.open(\'/tmp/yam_dump.yaml\', \'w\') '
+            '{|f| f.write(Settings.to_hash.deep_stringify_keys.to_yaml) }"'
+        )
+        if writeout.rc:
+            logger.error("Config couldn't be found")
+            logger.error(writeout.output)
+            raise Exception('Error obtaining config')
+        base_data = self.ssh_client.run_command('cat /tmp/yam_dump.yaml')
```
Comment (izapolsk) [inline utils/appliance/__init__.py]: OPTIONAL: we have the number of commands which we run over ssh everywhere. I'd create an SSHAgent with pre-defined commands

File: utils/appliance/__init__.py (hunk 1 above)
Comment (psav) [inline utils/appliance/__init__.py]: I'll leave as is as for now there may be something relying on this exception

File: utils/appliance/__init__.py (hunk 2 above)
Comment (psav) [inline utils/appliance/__init__.py]: I think we could, not sure if we'd just end up with a bunch of mishmash commands. It doesn't seem too horrible here to keep this......but I'll definitely consider it

**Pull request merged**